vnext/ReactUWP/Views/DynamicAutomationProperties.h

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <winrt/Windows.UI.Xaml.h>
+
+#include "cppwinrt/DynamicAutomationProperties.g.h"
+
+namespace winrt::react::uwp::implementation
+{
+
+struct DynamicAutomationProperties : DynamicAutomationPropertiesT<DynamicAutomationProperties>
```

Comment:
Not just specific to this type, but I feel like a lot of the new types you've added here could use some top level "this is what this class is for" type comment blocks.

Reply:
REsolved.